metadata/metadata_service.go

```
@@ -298,11 +358,14 @@ func (mtsV2 *MetadataServiceV2) GetMetadataInfo(ctx context.Context, req *triple
 	if err != nil {
 		return nil, err
 	}
+	if metadataInfo == nil {
+		return nil, nil
+	}
```

Silent failure is not good.

---

sounds good